google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -432,6 +432,7 @@ def delete force: nil
         #   length is 1,024 characters.
         # @param [String] name A descriptive name for the table.
         # @param [String] description A user-friendly description of the table.
+        # @param [EncryptionConfiguration] encryption_configuration A configuration for custom table data encryption.
```

Comment:
@tswast This can be broken up as 
# @param [EncryptionConfiguration] encryption_configuration A configuration for 
#   custom table data encryption.

Reply:
`# @param [EncryptionConfiguration] encryption_configuration A configuration for
#   custom table data encryption.`

Reply:
Not sure how to do this in github, just indented with 3 spaces after the pound sign

Reply:
Use three backticks to demarcate a multi-line code block (add "ruby" after the opening backticks as a hint for code highlighting):

<pre>
```ruby
        # @param [EncryptionConfiguration] encryption_configuration A configuration for 
            custom table data encryption.
```
</pre>